### PR TITLE
Added the ability to get all counters

### PR DESCRIPTION
**Pull request by viktor-liablin**
Added the ability to get all counters

### CODE REVIEW

File: src/urls.js [hunk 1]
```diff
@@ -154,7 +154,8 @@ export const apiDocsService = (appId, serviceId) => `${apiDocs(appId)}/services/
 export const apiDocsGeo = appId => `${apiDocs(appId)}/geo`
 
 export const cache = (appId, key) => `${appConsole(appId)}/cache${optional(key)}`
-export const counters = (appId, key) => `${systemData(appId)}/counters${optional(key)}`
+export const counters = (appId, pattern) => `${appConsole(appId)}/counters/${pattern || '*'}/list`
```
Comment (stanislav-grin) [inline src/urls.js:157]: ```export const counters = (appId, pattern = '*') => `${appConsole(appId)}/counters/${pattern}/list``` ?

Comment (viktor-liablin) [reply src/urls.js]: \+

Comment (Valodya) [reply src/urls.js]: I assume the first variant was better, for, instance if you run the method with the following arguments `counters('myApp', null)`  it returns the next result: `/counters/null/list`

Comment (viktor-liablin) [reply src/urls.js:157]: \+

**Pull request merged**